enterprise/websvc.rest/licenseinfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+  <licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/DefaultResource.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/GenericItemResource.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/GenericResource.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/JsEntity.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/JsProjectStub.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/JsReadme.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/JsStub.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/JsStubSupport.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/JsTestStubs.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/PersistenceService.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/PersistenceServiceNoJTA.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/PersistenceServiceStatic.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/RefConverter.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/RestConnection.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/RestProxyServlet.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/codegen/resources/UriResolver.template</file>
+        <file>src/org/netbeans/modules/websvc/rest/wizard/resources/JerseyClient.template</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/websvc/rest/resources/rest-build.xsl</file>
```

Comment:
This is an XSTL file, that is used to generate the build file. It is not a template and can hold the ASF header without a problem. Please add it to the XML file itself and remove it here.